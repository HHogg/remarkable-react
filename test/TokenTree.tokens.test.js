import buildTokenTree from './buildTokenTree';

describe('TokenTree tokens', () => {
  test('custom', () => {
    expect(buildTokenTree(`

% Lorem

      `, {}, {
        components: {
          video: 'video',
        },
        tokens: {
          video_token: 'video',
        },
      })).toMatchSnapshot();
  });
});
