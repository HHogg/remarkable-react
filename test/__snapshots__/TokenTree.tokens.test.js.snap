// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TokenTree tokens custom 1`] = `
Array [
  Object {
    "children": Array [
      Object {
        "children": undefined,
        "props": Object {},
        "type": "video",
      },
    ],
    "props": Object {},
    "type": "p",
  },
]
`;
